train.py

```diff
@@ -19,11 +22,13 @@
 IMAGE_WIDTH = 304
 TARGET_HEIGHT = 55
 TARGET_WIDTH = 74
-
+CHECKPOINT_PATH = 'checkpoints/ckpt'
```

Comment:
can we change this to COARSE_CHECKPOINT_PATH, and change the path to checkpoint/coarse/coarse_ckpt?

also, update CHECKPOITN_DIR to COARSE_CHECKPOINT_DIR

Reply:
updated and chnaged save freq to 1 epich